contributing-guides/style-guide.md

```diff
@@ -67,3 +67,14 @@ Keep the following guidelines in mind when choosing tokens:
 
 In general, tokens should make it as intuitive as possible
 to figure out how to use the command and fill it in with values.
+
+
+More technical wording on description lines should use the `backtick` syntax.
+Use backticks on the following:
+
+1. Filenames, ex. `package.json`.
```

Comment:
How about merging 1 and 2 togerther into "Paths"?

Reply:
IMO it is not clear that filenames would be included in "paths". 
I still believe we can merge them if we have two examples: a sole filename and a full path (directory+filename). What do you think?

Reply:
Good idea.  That must be the missing bit I was sensing.

Reply:
done